ui/src/services/segmenter/SegmenterStatus.js

```diff
@@ -1,13 +1,13 @@
 export const getSegmenterStatus = (segmenter) => {
   const statusMapping = {
     inactive: {
-      label: "inactive",
```

Comment:
Changing the case here so that the same map can be used for the status badge in the Segmenter details view as well as in the Segmenters list view, in capital case.